libbeat/scripts/Makefile

```diff
@@ -309,13 +309,9 @@ coverage-report:
 fields: mage
 	mage -v fields
```

Comment:
Can you set this line to `@mage fields`. I was debugging when changed it to its current state and didn't undo my changes.